### DISCUSSION TITLE
Sharing data from fixtures to the non test modules

### DISCUSSION THREAD

**Discussion by Puneeth1** [pytest-dev/pytest#11981] [Q&A]
what is the ideal way of Sharing data from fixtures to the non test modules ? 

Currently, I am calling a functions of non test modules in fixtures to share the data.

**Comment by RonnyPfannschmidt** [comment]
It's not clear what you mean by sharing data

In general the recommendation is to explicitly use resources as parameters to functions and context managers

Sharing data as some kind of magic global is generally a common source of hard to debug issues
